ecl/hthor/hthor.cpp

```diff
@@ -645,6 +644,7 @@ void CHThorDiskWriteActivity::close()
 {
     diskout.clear();
     outSeq.clear();
+    io.clear();
```

Comment:
This now means that the publish will not include the number of writes in the properties.

Reply:
right, will fix.